sources/dictionary/resource.yml

```diff
@@ -85,6 +86,7 @@ tabularDataResource:
     profile:
       "$ref": "#/definitions/profile"
       propertyOrder: 10
+      default: tabular-data-resource
```

Comment:
@georgiana-b I don't think we can add a default on TDP, as it means a package that is validated as TDP, which doesn't have a profile property, will pass validation anyway, but, we do require an explicit property for all profiles (not the base/default descriptors though)